Makefile

```diff
@@ -319,6 +320,16 @@ generate-go-conversions-kubeadm-control-plane: $(CONVERSION_GEN) ## Generate con
 		--output-file-base=zz_generated.conversion $(CONVERSION_GEN_OUTPUT_BASE) \
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
 
+.PHONY: generate-go-conversions-runtime
+generate-go-conversions-runtime: $(CONVERSION_GEN)
```

Comment:
Let's add this to the generate-go-conversions-core target, just like we generate deep copy etc for runtime sdk also in the core target